packages/@aws-cdk/aws-ecs/lib/base/task-definition.ts

```diff
@@ -464,6 +509,43 @@ export interface Host {
   readonly sourcePath?: string;
 }
 
+/**
+ * The port number as well as the protocol of a container host port.
+ *
+ * @internal
+ */
+export interface HostPort {
+  /**
+   * Port number of the container.
+   */
+  readonly port: number;
+
+  /**
+   * Protocol of the port.
+   */
+  readonly protocol: Protocol;
+}
+
+/**
+ * Properties for defining an ECS target.
+ */
+export interface LoadBalancerTargetOptions {
+  /**
+   * The name of the container.
+   */
+  readonly containerName: string;
+
+  /**
+   * The port number of the container.
```

Comment:
Mention that a port mapping for it must already have been created through `addPortMapping()`.

Can we make this optional and still do something useful? Like, take the first port mapping added if the value is not supplied?

Reply:
That's a good point. I think we can do that.